src/applications/mhv-landing-page/mocks/api/index.js

```diff
@@ -1,29 +1,49 @@
 /* eslint-disable camelcase */
 const delay = require('mocker-api/lib/delay');
 
-const commonResponses = require('../../../../platform/testing/local-dev-mock-api/common');
+const MOCK_TYPES = Object.freeze({
```

Comment:
Note: I got tired of switching the users, so I made this quick selector. More can be added and you change the mock type on the last line. I found no way to use command line arguments for this, and I did not want to generate different modules at this time.

Reply:
This looks like progress, regardless of our inability to pass cli arguments.